integration/bug-report-test.ts

```diff
@@ -37,28 +37,25 @@ beforeAll(async () => {
     ////////////////////////////////////////////////////////////////////////////
     files: {
       "app/routes/index.jsx": js`
-        import { json, useLoaderData, Link } from "remix";
+        import { json, useActionData, Form, Link } from "remix";
 
-        export function loader() {
-          return json("pizza");
+        export async function action({ request }) {
+          let formData = await request.formData();
+          return json(Object.fromEntries(formData));
         }
 
         export default function Index() {
-          let data = useLoaderData();
+          let actionData = useActionData();
           return (
-            <div>
-              {data}
-              <Link to="/burgers">Other Route</Link>
+            <div onClick={(event) => event.stopPropagation()}>
+              <pre>{JSON.stringify(actionData)}</pre>
+              <Form action="/?index" method="post">
+                <button type="submit" name="_action" value="add">Add</button>
```

Comment:
`name` can just be `action` instead

See f47a5114bb1a208f8316d1cc86eaee25699e7457 & #2164

```suggestion
                <button type="submit" name="action" value="add">Add</button>
```